attestation-manager/src/pkg/constants/config_test.go

```diff
@@ -0,0 +1,137 @@
+package constants
+
+import (
+	"os"
+	"testing"
+)
+
+// helper to set and unset env vars for each test
+func withEnv(env map[string]string, fn func()) {
+	originals := make(map[string]string)
+	for k, v := range env {
+		originals[k] = os.Getenv(k)
+		os.Setenv(k, v)
+	}
+	fn()
+	for k, v := range originals {
+		os.Setenv(k, v)
+	}
+	// Unset any env vars that were set in env but not present in originals (i.e., were not set before)
+	for k := range env {
+		if _, ok := originals[k]; !ok {
+			os.Unsetenv(k)
+		}
+	}
+}
+
+func validEnv() map[string]string {
+	return map[string]string{
```

Comment:
you only need these valid param from this function. wondering if you want to define this in heap ?